code/game/objects/items/toys.dm

```diff
@@ -1495,3 +1495,44 @@
 	if(cooldown <= world.time)
 		cooldown = (world.time + 10)
 		addtimer(CALLBACK(GLOBAL_PROC, .proc/playsound, src, 'sound/effects/blobattack.ogg', 50, FALSE), 0.5 SECONDS)
+
+
+/*
+ * Eldritch Toys
+ */
+
+/obj/item/toy/eldritch_book
+	name = "Codex Cicatrix"
+	desc = "A toy book that closely resembles the Codex Cicatrix. Covered in fake polyester human flesh and has a huge goggly eye attached to the cover. The runes are gibberish and cannot be used to summon demons... Hopefully?"
+	icon = 'icons/obj/eldritch.dmi'
+	icon_state = "book"
+	w_class = WEIGHT_CLASS_SMALL
+	attack_verb_continuous = list("sacrifices", "transmutes", "graspes", "curses")
+	attack_verb_simple = list("sacrifice", "transmute", "grasp", "curse")
+	/// Helps determine the icon state of this item when it's used on self.
+	var/book_state = FALSE
```

Comment:
This variable name is not descriptive, rename it to something that explains that it is whether the book is *open*, not just a "state", which doesn't even suggest a boolean.

Reply:
Funnily enough, it looks like you had it right before with `open`! 😄